app/components/pages/CustomFieldsSettings/tests/CreateEditCustomFieldDefaultValue.test.js

```
@@ -0,0 +1,149 @@
+      setDefaultValueError: setDefaultValueErrorMockFn,
+    },
+    customFieldMeta: {},
+    enumOptions: [],
+    ...props,
+  };
+
+  return render(<Component {...defaultProps} />);
+};
+
+describe('CreateEditCustomFieldDefaultValue', () => {
+  let errorMessage = 'Test error message';
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'errorMessage' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)